tcf_website/static/course/javascript/course_data.js

```diff
@@ -1,63 +1,97 @@
 const loadData = data => {
-    const { a_plus, a, a_minus, b_plus, b, b_minus, c_plus, c, c_minus, d_plus, d, d_minus, f, withdraw, drop } = data;
-    const grades_data = [a_plus, a, a_minus, b_plus, b, b_minus, c_plus, c, c_minus, d_plus, d, d_minus, f, withdraw, drop];
+    const { a_plus, a, a_minus, b_plus, b, b_minus, c_plus, c, c_minus, d_plus, d, d_minus, f, ot, drop, withdraw } = data;
```

Comment:
Considering that more students likely drop than withdraw, I think the graphs look better with the "last" slice of orange being thicker, as opposed to having a thin slice:

![image](https://user-images.githubusercontent.com/55100084/105950439-642e6b00-603c-11eb-8d6d-10b52350f115.png)

Would love to check how the old tCF did it...but the old site is now down.

Reply:
Also, I think the `ot` variable should be renamed to something like `other` (or maybe `pass`, as that's what it seems like it represents most of the time?) for clarity - it was a little ambiguous to me what it meant until I read further into the PR.

Reply:
1. `withdraw` wasn't there before, and I added that. I thought it was also a noteworthy piece of information, but let me know if you think I should remove it since I added it without asking the team. If not, I'll just change the order.
2. Yeah, I'll rename it. I just kept the way it is, but renaming it will definitely increase readability/maintainability. Also, the data files containing `OT` also uses `DR` for drop and `W` for withdraw, which means previous maintainers didn't always keep the original names in the data files.

Let me know whether I should discard `withdraw`, and I'll make the above changes in a single commit.

Reply:
Edit: Sorry, it seems like `withdraw` was already there and I only changed the order of  `drop` and `withdraw`.

Reply:
Renamed all instances of `ot` to `other` except when it's used for the name of the model (646a83b6593ab5a173f1512fdd39087731a0cfe8 and 12d305d05916269a7c47012416b85dc077d5c2f9). I'm not 100% sure, but I think changing a model field *might* break the database because we create migration files on the fly every time.

Reply:
Fixed in 97690bd601faf21beff4bdfc9ff3ff2fcbabc1d0 (color: `#6775A6`)